src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -361,14 +362,58 @@ invalidated when a device disconnects. This means your code should always
 retrieve (through `getPrimaryService(s)`, `getCharacteristic(s)`, etc.) these
 attributes after reconnecting.
 
+### Read and write to Bluetooth Descriptors
+
+Bluetooth GATT descriptors are defined attributes that describe a
+characteristic value. You can access them, read and write to in a similar way
```

Comment:
Change 'access them, read and write' to 'read and write them'. So the whole sentence would read: 'You can read and write them in a similar way to...'.